### PR TITLE
bpo-43472: Ensure PyInterpreterState_New audit events are raised when called through _xxsubinterpreters module

### PR DESCRIPTION
**Pull request by zooba**
https://bugs.python.org/issue43472

### CODE REVIEW

File: Modules/_xxsubinterpretersmodule.c [hunk 1]
```diff
@@ -2025,7 +2025,7 @@ interp_create(PyObject *self, PyObject *args, PyObject *kwds)
     }
 
     // Create and initialize the new interpreter.
-    PyThreadState *save_tstate = PyThreadState_Swap(NULL);
+    PyThreadState *save_tstate = PyThreadState_Get();
     // XXX Possible GILState issues?
```
Comment (zooba) [inline Modules/_xxsubinterpretersmodule.c:2029]: Maybe we can delete this comment now?

Comment (ericsnowcurrently) [reply Modules/_xxsubinterpretersmodule.c:2029]: I'm not sure the gilstate issues with subinterpreters have been resolved.

**Review by ericsnowcurrently — APPROVED** [review]
LGTM

File: Modules/_xxsubinterpretersmodule.c (hunk 1 above)
Comment (zooba) [reply Modules/_xxsubinterpretersmodule.c:2029]: Going to leave it here

**Pull request merged**